README.md

```diff
@@ -175,6 +169,15 @@ hoodie.plugin(function (hoodie) {
 })
 ```
 
+```js
+  const sayHi = require('sayHi')
+  const grabADrink = require('grabADrink')
+
+  hoodie
+    .plugin(sayHi)
+    .plugin(grabADrink)
+```
```

Comment:
can we add to the example that an object can be passed in, too? Maybe like this?

```js
  const sayHi = require('sayHi')
  const grabADrink = require('grabADrink')

  hoodie
    .plugin(sayHi)
    .plugin({
      grabADrink: grabADrink
    })
```